cmd/temporalite/mtls_test.go

```diff
@@ -158,3 +164,13 @@ func TestMTLSConfig(t *testing.T) {
 		t.Fatalf("Unexpected response %s, with body %s", res.Status, string(body))
 	}
 }
+
+func isUIPresent() bool {
```

Comment:
Nit: would you mind updating `ui_test.go` and `ui_disabled_test.go` files to use this function? This is the third copy of this bit of code.